main.tf

```diff
@@ -1,8 +1,11 @@
-// verify Domain ownership
-resource "aws_route53_record" "ownership" {
+# see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record
+resource "aws_route53_record" "main" {
   zone_id = var.zone_id
   name    = "_github-challenge-${var.github_organization}.${local.zone_name}"
```

Comment:
`${local.zone_name}`

This local seems redundant.
It is declared as `zone_name = data.aws_route53_zone.zone.name ` in variables.tf and only used once.
Not hugely impactful.

Reply:
Very redundant, as there's indeed just a single use of it. Great spot ❤️